README.rst

```diff
@@ -94,7 +94,7 @@ To use the action simply add the following lines to your ``.github/workflows/mai
         steps:
         - uses: actions/checkout@v1
         - name: Run salt-lint
-          uses: roaldnefs/salt-lint-action@master
+          uses: warpnet/salt-lint-action@master
```

Comment:
We might want to keep the GitHub action on my personal account as it might break the marketplace listing.

```suggestion
          uses: roaldnefs/salt-lint-action@master
```